### ISSUE TITLE
Train loss Nan

### ISSUE THREAD

**Issue by QingyuanLi1211** [lin-tan/CURE#19]
Dear author! When I train the model with the demo data and with my own data, I got the same result: 
1 epoch:1, load data:1200, lr:6.25e-05, loss:nan, apr_loss:nan, lm_loss:nan, time:6s, oom:101
val loss:nan, val apr_loss:nan, val lm_loss:nan, val ppl:nan, oom:9
...
the loss, apr_loss, and lm_loss are always nan. 
So, how could I solve this problem?

**Comment by BEbillionaireUSD** [comment]
Hi, could you please share how you solve this issue?

**Comment by salted-yu** [comment]
@HanJin996 Hi, could you please share how you solve this issue?

**Comment by QingyuanLi1211** [comment]
Sorry, dear friends, I wasn't able to solve the problem either, and I ended up abandoning the tool.

**Comment by salted-yu** [comment]
Thanks for your reply.
